### DISCUSSION TITLE
Should changing mirror.linkerd.io/exported to false from true remove the mirrored service?

### DISCUSSION THREAD

**Discussion by NeckBeardPrince** [linkerd/linkerd2#8468] [Help]
I have my Elasticsearch StatefulSet service with `mirror.linkerd.io/exported: true` sitting in cluster A and I can see it being mirrored in cluster B. When I change the label to "false" I would've thought the service-mirror in cluster B would pick that up and remove the mirrored service. But is that not the case? I only started using Linkerd last week, so if there is something I'm not understanding, please let me know. I have `-enable-headless-services` enabled as well.

```
Client version: stable-2.11.2
Server version: stable-2.11.2
```

**Comment by kleimkuhler** [comment]
Sorry for the late response on this, but this was an issue and has been fixed by #8734.
